backend/lib/processor.py

```diff
@@ -541,7 +575,7 @@ def write_csv_items_and_finish(self, data):
 		self.dataset.update_status("Finished")
 		self.dataset.finish(len(data))
 
-	def write_archive_and_finish(self, files, num_items=None, compression=zipfile.ZIP_STORED):
+	def write_archive_and_finish(self, files, num_items=None, compression=zipfile.ZIP_STORED, finish=True):
```

Comment:
This doesn't seem to be used anywhere (is finish ever `False`?)

Reply:
Ah, that's actually from the `tracker_tracker` branch, BUT will be needed if we have any datasources that are zip archives on the master branch (and when I manage to fully extract `tracker_tracker` into an extensions repo). https://github.com/digitalmethodsinitiative/4cat/blob/ffa5676ac0ec0a73de009814bcf3620c3484c291/backend/lib/search.py#L356 It does not do anything without the modifications to `search.py`. 

I guess it snuck in here when I was testing this branch on the tracker_tracker branch. I can remove it and include it on the eventual PR for needed changes in order to allow Selenium based datasources and processors. And zip archive datasource types.

Reply:
Makes sense - I think it's better to leave it for the other PR then, so it's clear what this change does in context